src/Apps/ArtAdvisor/03-Instruction-Experiments/App.tsx

```diff
@@ -72,7 +75,9 @@ export const App: FC = () => {
             onChange={e => setUserInput(e.target.value)}
           />
           <Spacer x={4} />
-          <Button type="submit">Send</Button>
+          <Button type="submit" disabled={isGenerating}>
+            Send
+          </Button>
```

Comment:
Given the latency of our currently non-streaming responses, I want to prevent rage-clicking that would result in extra requests / token consumption.